src/components/BannerImage/BannerImage.tsx

```diff
@@ -1,11 +1,42 @@
-import { BannerContainer, Banner } from "./BannerImage.styles";
+import { IMainBanner } from "@/utils/interfaces";
+import { Icon } from "../Icon/Icon";
+import Image from "next/image";
+import { LinkButton } from "../LinkButton/LinkButton";
+
+const MainBanner = ({ content }: { content: { fields: IMainBanner } }) => {
+  console.log(content);
+  const { title, subtitle, image } = content.fields;
 
-const BannerImage = () => {
   return (
-    <BannerContainer>
-      <Banner src={"/banner.png"} />
-    </BannerContainer>
+    <div className="overflow-hidden relative flex justify-center items-center w-full mt-4 min-h-[450px]">
```

Comment:
```suggestion
    <div className="overflow-hidden relative flex justify-center items-center w-full lg:mt-4 min-h-[380px] lg:min-h-[510px]">
```
this margin on mobile appears like this, it's slightly different from the Figma design

![image](https://github.com/user-attachments/assets/f6c82fbd-2ed5-4511-99f7-4c65e3ec6181)